### PR TITLE
(maint) Add Razor server file layout

### PR DESCRIPTION
**Pull request by melissa**

### CODE REVIEW

**Comment by melissa** [comment]
Currently a work in progress as I update the file layout of razor

**Comment by melissa** [comment]
cc/ @turbodog and @smcclellan can you review the razor file layout outlined here?

File: file_paths.md [hunk 1]
```diff
@@ -419,6 +420,43 @@ The package will install a service named `puppetmaster`, create a
...
+            razor-server                                                  /opt/razor
+                bin
+                    razor-admin
+                    razor-binary-wrapper
+                brokers
+                lib
+                razor-server.env                                          /usr/share/razor-server/razor-server.env
+                shiro.ini
+                vendor
+            torquebox                                                    /opt/razor-torquebox
```
Comment (smcclellan) [inline file_paths.md]: Super minor nit: Does whitespace matter here?

Comment (melissa) [inline file_paths.md]: just in terms of human readability, so yes, it does. I'd like to add explanations of these directories in addition to their previous location. See above layouts for examples

Comment (kylog) [inline file_paths.md]: I'm guessing that @smcclellan was pointing out that `/opt/razor-torquebox` needs to be preceded by one more space to align with the fields above.

File: file_paths.md [hunk 2]
```diff
@@ -419,6 +420,43 @@ The package will install a service named `puppetmaster`, create a
...
             puppetmaster.pid
 
+# razor-server
+
+The package will install a service named `razor-server`, create a `razor`
+user and group, and run the service as the `razor` user.
+
+    /etc/puppetlabs/razor-server                                          /etc/razor
+        config.yaml
+        razor-toequebox.sh
```
Comment (kylog) [inline file_paths.md]: s/toeque/torque/ ?

File: file_paths.md (hunk 1 above)
Comment (melissa) [inline file_paths.md]: Oooh! I didn't even notice that! Yes, I'll fix that

File: file_paths.md (hunk 2 above)
Comment (melissa) [inline file_paths.md]: Fixed!

File: file_paths.md (hunk 1 above)
Comment (melissa) [inline file_paths.md]: Fixed!

**Comment by smcclellan** [comment]
👍

**Comment by melissa** [comment]
cc/ @turbodog who else do we need to sign off on these pathing changes?

File: file_paths.md [hunk 3]
```diff
@@ -419,6 +420,44 @@ The package will install a service named `puppetmaster`, create a
...
+
+The package will install a service named `razor-server`, create a `razor`
+user and group, and run the service as the `razor` user.
+
+    /etc/puppetlabs/razor-server                                         /etc/razor
+        config.yaml
+        razor-torquebox.sh
+        shiro.ini
+    /opt/puppetlabs/bin
+        razor-admim                       -> /opt/puppetlabs/server/apps/razor-server/share/razor-server/bin/razor-binary-wrapper
```
Comment (smcclellan) [inline file_paths.md]: Just noticed a typo here: admim => admin

Comment (melissa) [inline file_paths.md]: Thanks! Fixed!

File: file_paths.md [hunk 4]
```diff
@@ -421,6 +422,45 @@ The package will install a service named `puppetmaster`, create a
...
+                lib
+                razor-server.env                                         /usr/share/razor-server/razor-server.env
+                shiro.ini
+                vendor
+            torquebox                                                    /opt/razor-torquebox
+                jboss
+                jruby
+        var/razor                                                        /var/lib/razor
+    /opt/puppetlabs/server/data/razor-server
+        repo                                                             /var/lib/razor/repo-store
```
Comment (melissa) [inline file_paths.md]: @smcclellan I've updated this to reflect the repo store path changes you implemented before 1.4.0 went out

**Comment by smcclellan** [comment]
@melissa I'm 👍 to merge this if you feel it's ready now.

**Comment by melissa** [comment]
it's been ready for a while, so feel free to merge away. I'm fairly sure I've updated all the changes we made prior to release

**Pull request merged**